wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -51,49 +51,46 @@ General
 -   **Use the same slider tick rate on every difficulty** as it is a property of the music rather than the mapping. However, lower difficulties may use lower tick rates to reduce accuracy requirements for newer players, providing they still follow the rhythm of the song. Using high tick rates purely to increase score/combo/difficulty is senseless.
 -   **Hyperdashes should not be used when the destination of the hyperjump is located near the left or right border of the play field.** This creates an uncomfortable movement as the catcher is forcibly stopped upon reaching the border of the playfield. Try to leave at least 16 osupixels of space between the end point of the hyperjump and the border of the play field, respectively at x:16 or x:496 at most.
 
-Skinning
---------
+### Skinning
 
-### Rules
+#### Rules
 
 -   **Custom catchers must be included in v2 skin format.** This is to ensure correct display on all skins. The required filenames are "fruit-catcher-idle.png", "fruit-catcher-kiai.png" and "fruit-catcher-fail.png".
 -   **Custom fruits must include all necessary elements and be colored in a scale of grey colors.** This is to ensure that your images are clearly defined and of acceptable quality. The needed elements can be found in the osu! wiki </wiki/Skinning_Catch_the_Beat> . Additionally, it is recommendable to use transparent elements for the overlays.
```

Comment:
Change this sentence `The needed elements can be found in the osu! wiki </wiki/Skinning_Catch_the_Beat> .` to `The needed elements can be found at [Skinning/osu!catch](/wiki/Skinning/osu!catch).`